polaris-react/src/components/Text/Text.tsx

```diff
@@ -38,7 +38,7 @@ const VariantFontWeightMapping: {[V in Variant]: FontWeight} = {
 };
 
 export interface TextProps {
-  /** Adjust horizontal alignment of text */
+  /** Adjust horizontal alignment of text when `as` prop is set to a block element */
```

Comment:
```suggestion
  /** Adjust horizontal alignment of text for block elements */
```

This might be too brief? What do you think @aveline

Reply:
I like brevity! But I think a clear link to the `as` prop is worth calling out?